src/main/scala/play/api/libs/mailer/MailerPlugin.scala

```diff
@@ -273,3 +281,40 @@ case class AttachmentData(name: String,
                           mimetype: String,
                           description: Option[String] = None,
                           disposition: Option[String] = None) extends Attachment
+
+case class SMTPConfiguration(host: String,
+                             port: Int,
+                             ssl: Boolean = false,
+                             tls: Boolean = false,
+                             user: Option[String],
+                             password: Option[String],
+                             debugMode: Boolean = false,
+                             timeout: Option[Int] = None,
+                             connectionTimeout: Option[Int] = None)
+
+object SMTPConfiguration {
```

Comment:
> I would have fixed Configuration.from to do this for you, but there's a lot of code that depends on that where automatically adding the reference configuration would break it.

I totally understand that, I will use `Configuration.reference ++ Configuration.from()`, thanks!